libethereum/TransactionQueue.cpp

```diff
@@ -31,7 +31,28 @@ using namespace dev::eth;
 const char* TransactionQueueChannel::name() { return EthCyan "┉┅▶"; }
 const char* TransactionQueueTraceChannel::name() { return EthCyan " ┅▶"; }
 
-ImportResult TransactionQueue::import(bytesConstRef _transactionRLP, ImportCallback const& _cb, IfDropped _ik)
+TransactionQueue::TransactionQueue(unsigned _limit, unsigned _futureLimit):
+	m_current(PriorityCompare { *this }),
+	m_limit(_limit),
+	m_futureLimit(_futureLimit)
+{
+		unsigned verifierThreads = std::max(thread::hardware_concurrency(), 3U) - 2U;
+		for (unsigned i = 0; i < verifierThreads; ++i)
+			m_verifiers.emplace_back([=](){
+				setThreadName("txcheck" + toString(i));
+				this->verifierBody();
+			});
+}
+
+TransactionQueue::~TransactionQueue()
+{
+	m_aborting = true;
```

Comment:
The read of m_aborting in https://github.com/ethereum/cpp-ethereum/pull/2386/files#diff-37a6e0c4c4565fc12c93ad4e9f6445f6R381